includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -230,6 +233,85 @@ public function validate_statement_descriptor( $value, $request, $param ) {
 		return true;
 	}
 
+	/**
+	 * Validate the business support email.
+	 *
+	 * @param string          $value The value being validated.
+	 * @param WP_REST_Request $request The request made.
+	 * @param string          $param The parameter name, used in error messages.
+	 * @return true|WP_Error
+	 */
+	public function validate_business_support_email_address( string $value, WP_REST_Request $request, string $param ) {
+		$string_validation_result = rest_validate_request_arg( $value, $request, $param );
+		if ( true !== $string_validation_result ) {
+			return $string_validation_result;
+		}
+
+		if ( empty( $value ) ) {
+			return new WP_Error(
+				'rest_invalid_pattern',
+				__( 'Error: Support email address is required!', 'woocommerce-payments' )
+			);
+		}
+
+		if ( ! is_email( $value ) ) {
```

Comment:
Similarly to other methods in this class, we should merge the f=both ifs into one.

Reply:
Other methods validate only if there is a passed value as they allow empty values. Email setting is mandatory, and that's why it returns two separate error messages.